src/components/members/delete.tsx

```
@@ -1,27 +1,80 @@
-            </div>
-        </div>
-    )
+export default function Delete() {
+  const { closeModal, selectedMember, refreshUsers } = useContext(MemberContext)
+  const [user, loading, error] = useAuthState(auth)
+  
+  const onDelete = async () => {
+    await UserService.deleteUser(selectedMember?.uid || '', user)
+    closeModal()
+    setTimeout(() => {
+      refreshUsers && refreshUsers()
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)